oauthproxy.go

```diff
@@ -303,6 +304,15 @@ func NewOAuthProxy(opts *options.Options, validator func(string) bool) (*OAuthPr
 
 	logger.Printf("Cookie settings: name:%s secure(https):%v httponly:%v expiry:%s domains:%s path:%s samesite:%s refresh:%s", opts.Cookie.Name, opts.Cookie.Secure, opts.Cookie.HTTPOnly, opts.Cookie.Expire, strings.Join(opts.Cookie.Domains, ","), opts.Cookie.Path, opts.Cookie.SameSite, refresh)
 
+	trustedIPs := ip.NewNetSet()
+	for _, ipStr := range opts.TrustedIPs {
+		if ipNet := ip.ParseIPNet(ipStr); ipNet != nil {
+			trustedIPs.AddIPNet(*ipNet)
+		} else {
+			panic(fmt.Sprintf("Could not parse IP network (%s)", ipStr))
```

Comment:
This method can now return an error, so I think we should do that here rather than panicking (this was added recently hence the late change). Will have the same effect but is more graceful 🙂 
```suggestion
			return nil, fmt.Errorf("Could not parse IP network (%s)", ipStr)
```